### PR TITLE
Issue #9746: update AbstractClassNameCheckTest

### PR DESCRIPTION
**Pull request by Ritik2604**
fixes Issue #9746:
done some basic changes in AbstractClassNameCheckTest in order to use unique input files for each test methods.

### CODE REVIEW

**Comment by Ritik2604** [comment]
@timurt @romani @nmancus1 @pbludov whosoever is free please review this.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/abstractclassname/InputAbstractClassNameFormerFalsePositive.java [hunk 1]
```diff
@@ -1,12 +1,27 @@
 package com.puppycrawl.tools.checkstyle.checks.naming.abstractclassname;
+/* Config:
+ * default
+ *
+ */
```
Comment (timurt) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/abstractclassname/InputAbstractClassNameFormerFalsePositive.java:5]: it is a minor request but could you reformat inputs in the following way

```
package com.puppycrawl.tools.checkstyle.checks.naming.abstractclassname;

/*
 * Config: default
 */
abstract public class Inpu....
```

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/abstractclassname/InputAbstractClassNameFormerFalsePositive.java [hunk 2]
```diff
@@ -1,12 +1,27 @@
...
+abstract class AbstractClassOtherFP { // ok
+    abstract class NonAbstractInnerClassFP { // violation
     }
+}
 
-    class AbstractClass {
+class NonAbstractClassFP { // g
+}
+
+//^Abstract.*$ --> abstract
```
Comment (timurt) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/abstractclassname/InputAbstractClassNameFormerFalsePositive.java:20]: do we need these comments?

**Review by timurt — CHANGES_REQUESTED** [review]

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/abstractclassname/InputAbstractClassNameFormerFalsePositive.java (hunk 1 above)
Comment (Ritik2604) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/abstractclassname/InputAbstractClassNameFormerFalsePositive.java:5]: Yeah I will

**Pull request closed without merging**